utils/plotly_html_renderer.py

```
@@ -0,0 +1,186 @@
+try:
+    import plotly.graph_objects as go
+except ImportError:
+    go = None  # type: ignore
+
+logger = logging.getLogger(__name__)
+
+
+def plotly_figure_to_html(
+    fig: Optional["go.Figure"] = None,
+    div_id: Optional[str] = None,
+    include_plotlyjs: Union[str, bool] = "cdn",
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)